usmqe_tests/conftest.py

```diff
@@ -57,14 +57,51 @@ def logger_testcase(request):
 @pytest.fixture(
     params=[{
         "name": "Tom Hardy",
-        "username": "thardy",
-        "email": "thardy@tendrl.org",
+        "username": "tom-admin",
+        "email": "tom-admin@tendrl.org",
+        "role": "admin",
+        "password": "pass1234",
+        "email_notifications": True}])
+def valid_admin_user_data(request):
+    """
+    Generate valid data that can be imported into tendrl as a new user with
+    admin role.
+
+    ``params`` parameter takes list of dictionaries where each dictionary
+        contains ``username`` and ``password`` as keys.
+    """
+
+    return request.param
+
+
+@pytest.fixture
+def valid_new_admin_user(valid_admin_user_data):
+    """
+    Create user from valid_admin_user_data fixture and return these data.
+    At the end remove this user.
+    """
+    auth = login(
+        pytest.config.getini("usm_username"),
+        pytest.config.getini("usm_password"))
+    admin = tendrlapi_user.ApiUser(auth=auth)
+    admin.add_user(valid_admin_user_data)
+    yield valid_admin_user_data
+    admin.del_user(valid_admin_user_data["username"])
+    logout(auth=auth)
+
+
+@pytest.fixture(
+    params=[{
+        "name": "Tom Hardy",
+        "username": "tom-normal",
+        "email": "tom-normal@tendrl.org",
         "role": "normal",
         "password": "pass1234",
         "email_notifications": True}])
-def valid_user_data(request):
+def valid_normal_user_data(request):
```

Comment:
I think it would be better to use values which differs from the admin user in every field here (including name and password).